src/test/java/org/openrewrite/java/migrate/guava/NoGuavaPrimitiveAsListTest.java

```diff
@@ -0,0 +1,236 @@
+        //language=java
+        rewriteRun(
+          java(
+            """
+              import com.google.common.primitives.Floats;
+              import java.util.List;
+              
+              class Test {
+                  List<Float> floats = Floats.asList(1f, 2f);
+              }
+              """,
+            """
+              import java.util.Arrays;
+              import java.util.List;
+              
+              class Test {
+                  List<Float> floats = Arrays.asList(1f, 2f);
+              }
+              """
+          )
+        );
+    }
+
+    @Test
+    void replaceLong() {
+        //language=java
+        rewriteRun(
+          java(
+            """
+              import com.google.common.primitives.Longs;
+              import java.util.List;
+              
+              class Test {
+                  List<Long> longs = Longs.asList(1L, 2L);
+              }
+              """,
+            """
+              import java.util.Arrays;
+              import java.util.List;
+              
+              class Test {
+                  List<Long> longs = Arrays.asList(1L, 2L);
+              }
+              """
+          )
+        );
+    }
+
+    @Test
+    void replaceInt() {
+        //language=java
+        rewriteRun(
+          java(
+            """
+              import com.google.common.primitives.Ints;
+              import java.util.List;
+              
+              class Test {
+                  List<Integer> ints = Ints.asList(1, 2);
+              }
+              """,
+            """
+              import java.util.Arrays;
+              import java.util.List;
+              
+              class Test {
+                  List<Integer> ints = Arrays.asList(1, 2);
+              }
+              """
+          )
+        );
+    }
+
+    @Test
+    void replaceShort() {
+        //language=java
+        rewriteRun(
+          java(
+            """
+              import com.google.common.primitives.Shorts;
+              import java.util.List;
+              
+              class Test {
+                  List<Short> shorts = Shorts.asList((short) 1, (short) 2);
+              }
+              """,
+            """
+              import java.util.Arrays;
+              import java.util.List;
+              
+              class Test {
+                  List<Short> shorts = Arrays.asList((short) 1, (short) 2);
+              }
+              """
+          )
+        );
+    }
+
+    @Test
+    void replaceByte() {
+        //language=java
+        rewriteRun(
+          java(
+            """
+              import com.google.common.primitives.Bytes;
+              import java.util.List;
+              
+              class Test {
+                  List<Byte> shorts = Bytes.asList((byte) 1, (byte) 2);
+              }
+              """,
+            """
+              import java.util.Arrays;
+              import java.util.List;
+              
```

Comment:
Couple more instances here of trailing whitespace that might then trip up the formatter on a next pass through. I might just have to write a recipe to clear these out. 😅 
```suggestion
```

Reply:
I think I already resolved those in https://github.com/openrewrite/rewrite-migrate-java/pull/527/commits/0eb99f393d2a8396721a0557dbd4a01ab49f3b19 ?